packages/components/src/internal/components/domainproperties/DomainForm.spec.tsx

```diff
@@ -959,5 +959,20 @@ describe('DomainForm', () => {
         expect(form.find('.domain-field-row').length).toEqual(0);
         expect(form.find('.table-responsive').length).toEqual(1);
         expect(form.find('.domain-field-toolbar').length).toEqual(2);
+        expect(form.text()).not.toContain("Is Primary Key");
+    });
+
+    test('with summaryViewMode isPrimaryKey column', () => {
```

Comment:
Unfortunately setting state with a new `domain` that includes `domainKindName:'VarList'` in the previous test and attempting forcing a re-render does not work. 

I also only included 'VarList' and not 'IntList' as well because I thought that would be overzealous, but let me know what you think and I could add it as well!

Reply:
Since the if statement in the model.tsx file does include both of those types. I'd probably add both test cases, even if they are mostly copies of one another.

Reply:
Done!